app/org/sagebionetworks/bridge/redis/AbstractRedisKey.java

```diff
@@ -2,12 +2,21 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
+import java.util.EnumSet;
+
+import org.sagebionetworks.bridge.config.BridgeConfig;
+import org.sagebionetworks.bridge.config.BridgeConfigFactory;
+import org.sagebionetworks.bridge.config.Environment;
+
 abstract class AbstractRedisKey implements RedisKey {
 
+    private final BridgeConfig config =  BridgeConfigFactory.getConfig();
+    private final EnumSet<Environment> SHARED_ENV = EnumSet.of(Environment.DEV, Environment.LOCAL);
```

Comment:
We create a lot of Redis keys. We should make these static final, so they won't be re-instantiated for every Redis key we create.